net-p2p/automatic/files/automatic-0.9.0-util_test-header.patch

```diff
@@ -0,0 +1,10 @@
+--- ../src/tests/utils_test.c.org	2022-08-21 11:39:01.502847479 +0300
```

Comment:
Submitted upstream? If so, link?

Reply:
From previous thread:
"Reworked as patch. PR to upstream is here https://github.com/1100101/Automatic/pull/20"

Reply:
There's still no link at the top of the patch.

Reply:
Link added